### PR TITLE
Update Contributing section to include template model

### PR DESCRIPTION
**Pull request by carrodher**
##### Description of the change
Update Contributing section in README to include template model

##### Benefits
Adjust readme to new issues and pull requests approach

##### Possible drawbacks
Nothing

### CODE REVIEW

**Comment by sameersbn** [comment]
lgtm!

File: README.md [hunk 1]
```diff
@@ -428,17 +428,19 @@ To restore your application using backed up data simply mount the folder with Wo
...
-- Docker version (`$ docker version`)
-- Output of `$ docker info`
-- Version of this container (`$ echo $BITNAMI_IMAGE_VERSION` inside the container)
-- The command you used to run the container, and any relevant output you saw (masking any sensitive information)
+## Pull request
+You can submit a [pull request](https://github.com/bitnami/bitnami-docker-wordpress/pulls) with your contribution. Be sure to include the required information in the template that will appear submitting the pull request.
+
+We will try to test and integrate the change as soon as we can, but be aware we have many GitHub repositories to manage and we can't immediately respond to every request.
+
+Also don't be worried if the request is closed or not integrated sometimes the priorities of Bitnami might not match the priorities of the pull request. Don't fret, the open source community thrives on forks and GitHub makes it easy to keep your changes in a forked repo.Â
```
Comment (sameersbn) [inline README.md]: there is a fancy character let at the end of the sentence.

Comment (carrodher) [inline README.md]: Done!

File: README.md [hunk 2]
```diff
@@ -428,17 +428,19 @@ To restore your application using backed up data simply mount the folder with Wo
 
 # Contributing
 
-We'd love for you to contribute to this container. You can request new features by creating an [issue](https://github.com/bitnami/bitnami-docker-wordpress/issues), or submit a [pull request](https://github.com/bitnami/bitnami-docker-wordpress/pulls) with your contribution.
+We'd love for you to contribute to this container. You can do it by creating an [issue](https://github.com/bitnami/bitnami-docker-wordpress/issues) or submitting a [pull request](https://github.com/bitnami/bitnami-docker-wordpress/pulls) with your contribution.
```
Comment (sameersbn) [inline README.md]: Please use relative link to create a new issue with the markdown `[issue](../../issues/new)` 
 - sample: create a [new issue](../../issues/new)

Similarly for pull-request link use the markdown `[pull request](../../pulls)`
 - sample: create a [pull request](../../pulls)

File: README.md [hunk 3]
```diff
@@ -428,17 +428,19 @@ To restore your application using backed up data simply mount the folder with Wo
...
 
-# Issues
+## Issues
+If you encountered a problem running this container or you want request new features, you can file an [issue](https://github.com/bitnami/bitnami-docker-wordpress/issues).
```
Comment (sameersbn) [inline README.md]: Lets link directly to the new issue page and use relative links `[new issue](../../issues/new)`

File: README.md [hunk 4]
```diff
@@ -428,17 +428,19 @@ To restore your application using backed up data simply mount the folder with Wo
...
-If you encountered a problem running this container, you can file an [issue](https://github.com/bitnami/bitnami-docker-wordpress/issues). For us to provide better support, be sure to include the following information in your issue:
+If you suspect your issue is a bug, please edit your issue description to include the BUG REPORT INFORMATION in the template that will appear creating the issue.
 
-- Host OS and version
-- Docker version (`$ docker version`)
-- Output of `$ docker info`
-- Version of this container (`$ echo $BITNAMI_IMAGE_VERSION` inside the container)
-- The command you used to run the container, and any relevant output you saw (masking any sensitive information)
+## Pull request
+You can submit a [pull request](https://github.com/bitnami/bitnami-docker-wordpress/pulls) with your contribution. Be sure to include the required information in the template that will appear submitting the pull request.
```
Comment (sameersbn) [inline README.md]: similarly `[pull request](../../pulls)`

correction:
`will appear submitting` -> `will appear while submitting`

**Comment by carrodher** [comment]
Closed due to this was tests before implement the current templates

**Pull request closed without merging**